src/us/kbase/workspace/test/workspace/WorkspaceTester.java

```diff
@@ -1140,41 +1138,33 @@ protected void getNonExistantObject(WorkspaceUser foo, ObjectIdentifier oi,
 			new ResolvedWorkspaceID(1, "foo", false, false);
 	
 	protected void testObjectIdentifier(String goodId) {
-		new ObjectIdentifier(new WorkspaceIdentifier("foo"), goodId);
 		new ObjectIDResolvedWS(RWSID, goodId);
 //		new ObjectIDResolvedWSNoVer(fakews, goodId);
 		new ObjectIDNoWSNoVer(goodId);
 	}
 	
 	protected void testObjectIdentifier(String goodId, int version) {
-		new ObjectIdentifier(new WorkspaceIdentifier("foo"), goodId, version);
 		new ObjectIDResolvedWS(RWSID, goodId, version);
 //		new ObjectIDResolvedWSNoVer(fakews, goodId);
 		new ObjectIDNoWSNoVer(goodId);
 	}
 	
 	protected void testObjectIdentifier(int goodId) {
-		new ObjectIdentifier(new WorkspaceIdentifier("foo"), goodId);
 		new ObjectIDResolvedWS(RWSID, goodId);
 //		new ObjectIDResolvedWSNoVer(fakews, goodId);
 		new ObjectIDNoWSNoVer(goodId);
 	}
 	
 	protected void testObjectIdentifier(int goodId, int version) {
-		new ObjectIdentifier(new WorkspaceIdentifier("foo"), goodId, version);
 		new ObjectIDResolvedWS(RWSID, goodId, version);
 //		new ObjectIDResolvedWSNoVer(fakews, goodId);
 		new ObjectIDNoWSNoVer(goodId);
 	}
 	
-	protected void testObjectIdentifier(WorkspaceIdentifier badWS, String badId,
+	protected void testObjectIdentifier(
```

Comment:
All the deleted ObjectIdentifier tests below are now covered in the unit tests as they should be.